src/views/product-tutorials-view/helpers.ts

```diff
@@ -29,3 +35,10 @@ function sortAlphabetically(
     return 0
   }
 }
+
+export function filterAndSortCollections(
```

Comment:
Separated out the sorting from filtering, to be more functional for this use case.